pkg/controller/utils/capability_test.go

```diff
@@ -170,7 +170,7 @@ patch: {
 
 parameter: [string]: string
 `,
-			want: want{data: "{\"additionalProperties\":{\"type\":\"string\"},\"type\":\"object\"}", err: nil},
+			want: want{data: "{\"type\":\"object\"}", err: nil},
```

Comment:
why additionalProperties not exist?

Reply:
Cue 0.4.3 will parse `a: [string]:string` to `a:{}`.